mobile/lib/features/wallet/create_invoice_screen.dart

```diff
@@ -103,118 +104,115 @@ class _CreateInvoiceScreenState extends State<CreateInvoiceScreen> {
             FocusScope.of(context).requestFocus(FocusNode());
           },
           behavior: HitTestBehavior.opaque,
-          child: SafeArea(
-            child: Container(
-              constraints: const BoxConstraints.expand(),
-              child: Column(crossAxisAlignment: CrossAxisAlignment.stretch, children: [
-                const Center(
-                    child: Padding(
-                        padding: EdgeInsets.only(top: 25.0),
-                        child: Text(
-                          "What is the amount?",
-                          style: TextStyle(fontSize: 18.0, fontWeight: FontWeight.bold),
-                        ))),
-                Padding(
-                  padding: const EdgeInsets.all(32.0),
-                  child: Row(
-                    children: [
-                      Expanded(
-                        child: AmountInputField(
-                          value: amount != null ? amount! : Amount(0),
-                          hint: "e.g. ${formatSats(Amount(5000))}",
-                          label: "Amount",
-                          controller: _amountController,
-                          onChanged: (value) {
-                            if (value.isEmpty) {
-                              return;
+          child: ScrollableSafeArea(
```

Comment:
😆 This part of the diff is brutal when all that has likely changed is this bit at the top.

Reply:
I also removed the 

```
            child: Container(
              constraints: const BoxConstraints.expand(),
```

because it doesn't seem to do much really (neither before nor after this change...)